packages/dd-trace/src/opentracing/span.js

```diff
@@ -240,6 +240,7 @@ class DatadogSpan {
     }
 
     getIntegrationCounter('spans_finished', this._integrationName).inc()
+    this.setTag('_dd.integration', this._integrationName)
```

Comment:
Can we put this in the constructor instead? And just place the attribute directly into the span context `tags` instead of calling `setTag`